Crash upon empty drag & drop

### Current Behavior

When performing a drag & drop in the central info area of the main window, outside any loaded files, puddletag crashes with the following output:

    puddletag Version: 2.4.0
    Locale: en_US
    Traceback (most recent call last):
      File "/usr/lib/python3.12/site-packages/puddlestuff/tagmodel.py", line 1738, in dropEvent
        self.loadFiles(files, dirs, append=True)
      File "/usr/lib/python3.12/site-packages/puddlestuff/tagmodel.py", line 1852, in loadFiles
        assert files or dirs, 'Either files or dirs (or both) must be specified.'
               ^^^^^^^^^^^^^
    AssertionError: Either files or dirs (or both) must be specified.
    fish: Job 1, 'puddletag' terminated by signal SIGABRT (Abort)

![Screenshot_20241223_205920](https://github.com/user-attachments/assets/96afdc8c-0c9d-4873-8074-d3e41adfe8ca)


### Expected Behavior

The app shouldn't crash

### Steps To Reproduce

1. Open puddletag
2. Start and end a drag and drop inside the main info area, but outside any loaded files.
3. The app crashes

### Information about your system

Puddletag: 2.4.0
OS: CachyOS (Arch Linux)
Python: 3.12.7
PyQt: 5.15.11
Qt: 5.15.14

### Anything else?

_No response_

--- a/puddlestuff/tagmodel.py
+++ b/puddlestuff/tagmodel.py
@@ -1735,6 +1735,9 @@ class TagTable(QTableView):
                 else:
                     files.append(f)
 
+            if not dirs and not files:
+                return
+
             self.loadFiles(files, dirs, append=True)
 
     def dragMoveEvent(self, event):

